lua/shine/extensions/basecommands/server.lua

```diff
@@ -1259,10 +1268,10 @@ function Plugin:CreatePerformanceCommands()
 	AddAdditionalInfo( InterpCommand, "Interp", "ms" )
 
 	local function TickRate( Client, NewRate )
-		if NewRate > self.Config.MoveRate then
+		if NewRate < self.Config.MoveRate then
 			NotifyError( Client, "ERROR_TICKRATE_CONSTRAINT", {
 				Rate = self.Config.MoveRate
-			}, "Tick rate cannot be greater than move rate (%i).", true, self.Config.MoveRate )
+			}, "Tick rate must be greater than move rate (%i).", true, self.Config.MoveRate )
```

Comment:
Are you sure this is a must? The validator above allows them to be equal, so shouldn't this too?